tizen/src/constants.h

```diff
@@ -0,0 +1,83 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_CONSTANTS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_CONSTANTS_H_
+
```

Comment:
Why don't you wrap these constants up with a proper `namespace`.

Reply:
I wrapped them.